### DISCUSSION TITLE
Cannot load language file either remote or local

### DISCUSSION THREAD

**Discussion by fredemagi** [yajra/laravel-datatables#2869] [Q&A]
Hello,

I'm trying to make Yajra datatable work with different language files. However, no matter what I try, thye browser cannot load the resouce - either remote or local file. I have installed the newest version if Yajra datatables and Laravel 9. What do I do wrong? 

     //Setup ajax.
	$.ajaxSetup(
	{
		 headers: 
		 {
        	'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
    	 }
	});

 	$('#all-users-datatable').DataTable(
 	{
 		processing: true,
        serverSide: true,
        paging: true,
        ajax: '/get-users',
        language: {
        	url: '//cdn.datatables.net/plug-ins/1.10.19/i18n/Danish.json'
    	},
        columns: [
            { data: 'name' },
            { data: 'company' },
            { data: 'email' },
            { data: 'role' },
            { data: 'location' },
            { data: 'recent_login login' },
            {
                'data': 'action', 
                'name': 'action', 
                'orderable': true, 
                'searchable': true
            }
            ],
        columnDefs: [{
		    defaultContent: '-',
		    targets: '_all'
		  }]
    });

**Comment by Arne1303** [comment]
I think you should ask this in the datatables forum, laravel-datatables is only the backend.

Still I think your url is wrong, that file seems to only contain a small amount of translations. Try this one here:

`//cdn.datatables.net/plug-ins/1.12.1/i18n/da.json`

Got it from the datatables.net website: https://datatables.net/plug-ins/i18n/Danish#CDN
